README.md

```diff
@@ -12,65 +12,107 @@ Description
 
 Netlify hosted, Gatsby built, storyblok headless cms site for giving.stanford.edu
 
+## Installation
 
-Installation
----
+_Development_
 
-*Development*
 ```
-cp example.env.development .env.development
-* Manually add the access token to .env.development *
-nvm use
-npm install
-gatsby develop
+cp example.env .env
+* Manually add the vault secret and role id to .env
+* You can get the variables by running `netlify login` then `netlify link` and then `netlify env`
+* After the VAULT_ROLE_ID and VAULT_SECRET_ID environment variables have been set run
+npm run vault
+* Then fire up your development server using gatsby
+npm run dev
```

Comment:
```suggestion
npm run develop
```
Looks like Giving doesn't have `npm run dev` - please feel free to change `npm run develop` to `npm run dev` in the package.json file, @sherakama 😄  Less typing = good